src/components/organisms/ProjectsTable/ProjectsTable.jsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+// import Button from "../../atoms/Button";
+import ProjectTableRow from "../../molecules/ProjectsTableRow";
+import Projects from "./mockProjects";
+
+const Table = styled.table`
+  width: 100%;
+`;
+
+const ProjectsTable = () => {
+  return (
+    <Table>
+      <tr>
+        <th> Name </th>
+        <th> Description </th>
+        <th> Created At </th>
+      </tr>
```

Comment:
Need to wrap in <thead> to avoid html error
<img width="752" alt="image" src="https://github.com/JustArtur/task_tracker_react_2023/assets/79587920/1ea95814-0b07-46dd-ae62-15369119f9de">

```suggestion
      <thead>
        <tr>
          <th> Name </th>
          <th> Description </th>
          <th> Created At </th>
        </tr>
      </thead>
```